jupyterhub-k8s-network-tools.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: jupyterhub-k8s-network-tools
+  version: 3.3.8
+  epoch: 0
+  description: Network diagnostic tools for use within a JupyterHub Kubernetes cluster
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    runtime:
+      - iptables
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - name: Install
+    runs: |
+      mkdir -p "${{targets.destdir}}"
+
+test:
+  pipeline:
+    - name: Test iptables
+      runs: |
+        iptables --version
+
+update:
+  enabled: true
+  github:
```

Comment:
We're polling Git for updates, but theres no Git clone happening in this melange?

Reply:
We're polling github, just to tag the package correctly
There is nothing else to be used from the repository at all